lib/jellyfish/config_reader.ex

```diff
@@ -128,36 +128,45 @@ defmodule Jellyfish.ConfigReader do
     end
   end
 
-  def read_sip_config() do
-    sip_used? = read_boolean("JF_SIP_USED")
+  def read_components_used() do
+    components_used = System.get_env("JF_COMPONENTS_USED") || ""
+
+    components_used
+    |> String.split(" ", trim: true)
+    |> Enum.map(fn type ->
+      case Jellyfish.Component.parse_type(type) do
```

Comment:
I see that you have already got some "parser" implemented.
For future instead of parsing it "manually" and adding variant to case whenever new component is introduced you could just use `String.to_existing_atom` since every module name is an atom. 

The only possible drawback is that there is a case that one could add non-existing component. In that case you could just check if the component implements some behaviour 😄 But I wouldn't bother with that so much.